appserver/payara-appserver-modules/microprofile/openapi/src/main/java/fish/payara/microprofile/openapi/impl/processor/ApplicationProcessor.java

```diff
@@ -404,8 +407,8 @@ public void visitFormParam(FormParam param, AnnotatedElement element, ApiContext
 
             // Set the request body type accordingly.
             context.getWorkingOperation().getRequestBody().getContent()
-                    .get(javax.ws.rs.core.MediaType.WILDCARD).getSchema()
-                    .setType(formSchemaType);
+            .getMediaType(javax.ws.rs.core.MediaType.WILDCARD).getSchema()
```

Comment:
This is no standard formatting, could this be reverted?

Reply:
Sure. Not sure what happened there. IDE "helped" me or something.